mk/color.mk

```diff
@@ -0,0 +1,3 @@
+YELLOW=\033[0;33m
```

Comment:
It is not worthy to create a new file containing only 3 lines. Move these to `mk/common.mk`.